src/components/scene/meta-tags.js

```diff
@@ -0,0 +1,58 @@
+var register = require('../../core/component').registerComponent;
```

Comment:
I'm not sure if this should be a component since responsiveness is part of the core value proposition. Should we make it configurable? I 100% agree it should be a separate module to clean a-scene but not sure about it being a component. This could cause fragmentation of experiences if people remove the component of their scenes by accident or misunderstanding.